src/main.rs

```diff
@@ -1970,12 +2048,57 @@ fn cmd_check(
             // more clever comparisons.
             if store.config.exemptions != pruned_exemptions {
                 warn!("Your supply-chain has unnecessary exemptions which could be relaxed or pruned.");
-                warn!("Consider running `cargo vet prune` to prune unnecessary exemptions and imports.");
+                warn!("  Consider running `cargo vet prune` to prune unnecessary exemptions and imports.");
             } else if store.imports != pruned_imports {
                 warn!("Your supply-chain has unnecessary imports which could be pruned.");
-                warn!("Consider running `cargo vet prune` to prune unnecessary imports.");
+                warn!("  Consider running `cargo vet prune` to prune unnecessary imports.");
             }
         }
+
+        // Check for wildcard audits which will be expiring soon, and warn about them.
+        {
```

Comment:
Because the description of `locked` only mentioned that no new imported audits would be fetched. If it has a broader use case than the description implies I will change it to only check when not `--locked`.